Sources/NiftyMarkdownFormatter/NiftyMarkdownFormatter.swift

```diff
@@ -9,14 +9,18 @@ import SwiftUI
  - Parameter markdown: The text needed to be formatted, as a `String`
  */
 public struct FormattedMarkdown: View {
-    public init(markdown: String) {
+    public init(markdown: String, alignment: HorizontalAlignment? = nil, spacing: CGFloat? = nil) {
```

Comment:
Do you think it makes sense for the alignment to be an optional? Or should there be a provided default value for it?